Tools/TestWebKitAPI/Tests/JavaScriptCore/JSRunLoopTimer.mm

```diff
@@ -81,6 +82,7 @@ static void cycleRunLoop()
 
 TEST(JavaScriptCore, IncrementalSweeperMainThread)
```

Comment:
These tests were disabled in debug in [284925@main](https://commits.webkit.org/284925@main), so that will need to be reverted with this PR.